taxonomy/models.py

```diff
@@ -154,6 +156,62 @@ def __repr__(self):
         return '<CourseSkills id="{0}" skill="{1!r}">'.format(self.id, self.skill)
 
 
+class ProgramSkill(TimeStampedModel):
```

Comment:
How are we planning to populate this model with data?

Reply:
this model is ultimately planned to use in discovery which populates it with the results gathered from EMSI API

Reply:
Are we planning to get the skills from EMSI based on some program description?
Previously, we just aggregated the skills from the program constituent courses e.g. https://github.com/openedx/enterprise-catalog/blob/ff8264e662d4a32a550999f5dc463610f88d7276/enterprise_catalog/apps/catalog/algolia_utils.py#L633

Reply:
thanks @irfanuddinahmad for sharing the context here! 
the `ProgramSkill` model initially targets the degrees(aka programs) that would be ingested from 2U, the ingested degree/program data is different compared to existing edX programs.
you can find more details in this ADR PR: https://github.com/openedx/course-discovery/pull/3535/files